packages/synapse-react-client/src/lib/utils/theme/palette/Palettes.ts

```diff
@@ -22,6 +22,7 @@ export const generatePalette = (
     dark:
       (specificColors && specificColors[600]) ||
       tinycolor(mainColor).saturate(5).darken(4).toString(), // 600
+    contrastText: '#fff',
```

Comment:
All of our colors so far should use white, and some recent change caused many of them to use black (even in SWC). So let's just hard-code here and override if/when we need to

Reply:
|Before|After|
|---|---|
| <img width="67" alt="image" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/1d7a85fb-cdb7-479e-a4e6-8e8eeed66c29"> | |